src/style.css

```diff
@@ -0,0 +1,84 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+}
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
```

Comment:
```suggestion
```
let's not change flex-direction and use everywhere 'flex-basis'
Also add here (not in media-query, as you've written) `flex-wrap:wrap`